packages/now-build-utils/package.json

```diff
@@ -16,6 +16,10 @@
     "test-integration-once": "jest --env node --verbose --runInBand test/integration.test.js",
     "prepublishOnly": "./build.sh"
   },
+  "dependencies": {
+    "@now/frameworks": "^0.0.7",
+    "@now/routing-utils": "^1.5.2-canary.2"
+  },
```

Comment:
Maybe these should be `optionalDependencies` since they are only needed for TS?

Reply:
`optionalDependencies` are only really useful for packages that have a `postinstall` script that may fail. https://npm.github.io/using-pkgs-docs/package-json/types/optionaldependencies.html

Reply:
Also, they're not only for TS (just the compilation fails before runtime does).

Reply:
Since we publish with `ncc`, the dependencies are already bundled in the build and should be available at runtime.

This is really only necessary for TS types being exported.

Reply:
Ahh, true. I forgot about `ncc`. Well I'm down for either approach in that case. Though it's a bit weird that I need add/know about `@now/frameworks` when my Runtime exports a serverless function 🤔

Reply:
I already thing that `@now/build-utils` is doing too much so I think I would rather leave these out.

Maybe a prepublish script to copy the `.d.ts` would solve the TS issue.